smart-hadoop-support/smart-hadoop-client-3.1/src/main/java/org/smartdata/hadoop/filesystem/SmartFileSystem.java

```diff
@@ -171,6 +185,16 @@ public FileStatus getFileStatus(Path f) throws IOException {
             oldStatus.getOwner(), oldStatus.getGroup(),
             oldStatus.isSymlink() ? oldStatus.getSymlink() : null, oldStatus.getPath());
       }
+    } else {
```

Comment:
What if oldStatus=null? This else code block will throw NPE when oldStatus=null.